packages/event/src/Charcoal/Event/GenericEvent.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Charcoal\Event;
+
+use League\Event\HasEventName;
+
+/**
+ * Generic Event
+ */
+final class GenericEvent extends Event implements HasEventName
+{
+    /**
+     * @var mixed
+     */
+    private $subject;
+    private array $arguments = [];
```

Comment:
Unnecessary default value since one is assigned in the constructor.

```suggestion
    private array $arguments;
```